### DISCUSSION TITLE
SQLite Usage

### DISCUSSION THREAD

**Discussion by pavanbora1998** [invertase/react-native-firebase#7404] [Q&A]
Does firebase crashlytics uses the sqlite database? Because we got the VAPT issue (Use of Vulnerable SQLite Database and execution of Raw SQL Queries), after the implementation of crashlytics. If yes please let us know how to resolve this and provide the proper evidence for it.

**Comment by abhinavsharma1212** [comment]
Yes, I am also facing this issue, Please can anyone provide any answers?
